client/lib/post-normalizer/rule-pick-canonical-image.js

```diff
@@ -20,6 +20,7 @@ export default function pickCanonicalImage( post ) {
 		if ( canonicalImage ) {
 			canonicalImage = {
 				uri: canonicalImage.src,
+				src: canonicalImage.src,
```

Comment:
do we still need this? appears we're not using the canonicalImage at all in this patch?